### ISSUE TITLE
Credits, hat tips

### ISSUE THREAD

**Issue by elbotho** [serlo-org/PlantBuddies#3]
e.g.
* "Font Awesome by Dave Gandy - http://fontawesome.io".

**Comment by markus2330** [comment]
You might want to use https://reuse.software/ to indicate which parts use which license. But either way it would be great if you could select a FLOSS license.

**Comment by elbotho** [comment]
Thanks @markus2330. Are you just suggesting or are you interested in reusing something here?

**Comment by markus2330** [comment]
Yes, I am interested using relationships data+icons for https://github.com/ElektraInitiative/PermaPlant/

**Comment by markus2330** [comment]
Any update?

**Comment by elbotho** [comment]
Sorry, quite busy. 
For the plant/relationship data: 
- since it's tabular data I'm pretty sure we have no copyright on that so I can't license it (you are welcome to use it)
- If there should be something that is protected by copyright (notes/comments on the relationships) I hereby publish it under [CC-BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/deed.de)

The code can be used under [Apache-2.0 license](https://github.com/serlo/frontend/blob/staging/LICENSE).

I reached out to the illustrator regarding the icons and will get back to you on that. If she agrees those will be under [CC-BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/deed.de) as well.
